src/test/java/io/openliberty/tools/common/plugins/util/DevUtilTest.java

```diff
@@ -527,5 +530,14 @@ public void testOmitFilesList() throws Exception {
         assertEquals(omitFiles, util.getOmitFilesList(looseAppFile, new File("/src/main/webapp").getCanonicalPath()));
     }
 
-
+    @Test
+    public void testWriteDevcMetadataSimple() throws XMLStreamException, FactoryConfigurationError, IOException {
+        util.writeDevcMetadata();
+        File metaDataXml = new File(serverDirectory, "liberty-dev-metadata.xml");
+        assertTrue(metaDataXml.exists());
+        String content = FileUtils.readFileToString(metaDataXml, "UTF-8");
+        // assertTrue(content.contains("<startTime>"));
```

Comment:
Why is this commented out? If the `<stopTime>` below is not present, shouldn't the `<startTime>` be present? One or the other should always be there?